Training/Program.cs

```diff
@@ -1,7 +1,30 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+//Password validity checker which checks if the input password is valid or not
+Console.WriteLine ("Enter a password which satisfies the following guidelines" +
+   "\n1.Password must contain at least 6 characters" +
+   "\n2.Pasword should start with an uppercase letter" +
+   "\n3.Password must contain at least one lowercase letter" +
+   "\n4.Password must contain atleast one digit" +
+   "\n5.Password must contain at least one special character\n");
+for (; ; ) {                                                                                          // An infinite loop to ask the user to enter a valid password 
+   Console.Write ("Password: ");
+   string password = Console.ReadLine ();
+   if (StrongPassword (password) == "Accepted") {
+      Console.WriteLine (StrongPassword (password));                                                  // If there are no error messages, then accepts the password 
+      break;
+   } else
+      Console.WriteLine (StrongPassword (password));                                                  // Else asks for a valid password

```

Comment:
In both cases, you are printing the message that the function returns. Why are you checking the string here?

Reply:
Because, the method is returning a string "accepted". So, if the string "accepted" is retuned in the method, it means that the password has satisfied all of the conditions in the method and returns as strong password.  Thus, In order to print all the outputs of the method, i checked whether the output of the method is "accepted" or not.